docs/site/backends/docs-builder/pkg/hugo/commandeer.go

```diff
@@ -268,10 +270,19 @@ func (c *command) HugFromConfig(conf *commonConfig) (*hugolib.HugoSites, error)
 		// depsCfg := deps.DepsCfg{Configs: conf.configs, Fs: conf.fs, StdOut: c.hugologger.StdOut(), LogLevel: c.hugologger.Level()}
 		return hugolib.NewHugoSites(depsCfg)
 	})
+	h.Log = c.hugologger
+
 	return h, err
 }
 
-func (c *command) createLogger(running bool) (loggers.Logger, error) {
+func duration(key string, d time.Duration) slog.Attr {
+	return slog.Attr{
+		Key:   key,
+		Value: slog.StringValue(d.String()),
+	}
+}
+
+func (c *command) createLogger(running bool, logger log.Logger) (loggers.Logger, error) {
```

Comment:
we have in inside of c
use c.logger
```suggestion
func (c *command) createLogger(running bool) (loggers.Logger, error) {
```